utils/githooks/commit-msg.d/10-watermark

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-if ! grep 'Required-githooks: true' "$1"; then
+if grep -q '[:alpha:]' "$1" && ! grep -q 'Required-githooks: true' "$1"; then
```

Comment:
```suggestion
if grep -q '[[:alpha:]]*' "$1" && ! grep -q 'Required-githooks: true' "$1"; then
```

This has same effect for me, I can do :q! and it aborts but with your version, I get this warning:
grep: character class syntax is [[:space:]], not [:space:]

Reply:
Obviously, the warning seems unnecessary but nonetheless, it's annoying in the output.

Reply:
Thanks for the testing, Jeff. Looks like a difference in our grep programs---actually I recall having seen it when fixing the copyright updater in the past. Will fix.

We don't need the "*", right? The logic is that if there's any line with "at least one", not "zero or more", alphabet characters.